engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -70,4 +75,19 @@ package object engine {
     def toKey: ExecutionDatabaseKey = ExecutionDatabaseKey(execution.callFqn, execution.index.toIndex, execution.attempt)
     def executionStatus: ExecutionStatus = ExecutionStatus.withName(execution.status)
   }
+
+  //TODO: The `CromwellServer.workflowDescriptorMaterializerActor` referenced below should be injected rather than being used like this
```

Comment:
Although I might favour the former, workflowDescriptorMaterializerActor is actually MaterializeWorkflowDescriptorActor